plugins/flutter_aepedge/README.md

```diff
@@ -174,23 +174,101 @@ final ExperienceEvent experienceEvent = ExperienceEvent({
 });
 List<EventHandle> result = await Edge.sendEvent(experienceEvent);
 ```
+
+**Example with Datastream ID override**
+```dart
+Map<String, dynamic> xdmData = {"eventType": "SampleEventType"};
+final ExperienceEvent experienceEvent = ExperienceEvent({
+  "xdmData": xdmData,
+  "datastreamIdOverride": "SampleDatastreamId"
+});
+List<EventHandle> result = await Edge.sendEvent(experienceEvent);
+```
+
+**Example with Datastream confg override**
+```dart
+Map<String, dynamic> xdmData = {"eventType": "SampleEventType"};
+Map<String, dynamic> configOverrides = {"config": {
+      "com_adobe_experience_platform": {
+        "datasets": {
+          "event": {
+            "datasetId": "sampleDatasetID"
+          }
+        }
+      }
+    }};
+final ExperienceEvent experienceEvent = ExperienceEvent({
+  "xdmData": xdmData,
+  "datastreamConfigOverride": configOverrides
+});
+List<EventHandle> result = await Edge.sendEvent(experienceEvent);
+```
 ------
 ### Public classes
 #### ExperienceEvent
 Experience Event is the event to be sent to Adobe Experience Platform Edge Network. The XDM data is required for any Experience Event being sent using the Edge extension.
 
-##### Create Experience Event from Dictionary:
+You can create Experience Event either by using dictionaries or by utilizing convenience constructors.
 
+**Syntax**
 ```dart
-Map<String, dynamic> xdmData = {"eventType": "SampleEventType"};
+//Create Experience Event from Dictionary
+ExperienceEvent(this.eventData) 
+
+//Create Experience event using convenience constructor
+ExperienceEvent.createEventWithOverrides(final Map<String, dynamic> xdmData,
+    [final Map<String, dynamic>? data, final String? datastreamIdOverride, final Map<String, dynamic>? datastreamConfigOverride])
+ 
+```
+
+**Usage**
+##### Create Experience Event from Dictionaries:
+```dart
+// set free form data to the Experience event:
 final ExperienceEvent experienceEvent = ExperienceEvent({
-  "xdmData": xdmData
+  "xdmData": xdmData,
+  "data": data
+});
+
+// Set free form data and datastream id override to the current Experience event:
+final ExperienceEvent experienceEvent = ExperienceEvent({
+  "xdmData": xdmData,
+  "data": data,
+  "datastreamIdOverride": "sampleDatastreamId"
+});
+
+// Set free form data and datastream config override to the current Experience event:
+final ExperienceEvent experienceEvent = ExperienceEvent({
+  "xdmData": xdmData,
+  "data": data,
+  "datastreamConfigOverride": configOverrides
 });
 ```
 
-##### Add free form data to the Experience event:
+##### Create Experience event using convenience constructors:
```

Comment:
Can you update all the locations with this `Experience event` i.e. `event` starting with lower case `e`, like we have here